cluster-autoscaler/cloudprovider/gce/mig_info_provider.go

```diff
@@ -444,3 +446,18 @@ func (c *cachingMigInfoProvider) GetListManagedInstancesResults(migRef GceRef) (
 	c.cache.SetListManagedInstancesResults(migRef, listManagedInstancesResults)
 	return listManagedInstancesResults, nil
 }
+
+func createInstancesState(targetSize int64, actionsSummary *gce.InstanceGroupManagerActionsSummary) map[cloudprovider.InstanceState]int64 {
```

Comment:
We need a test for this method. I know there is currently no good public method to test this with, but in the meantime, please add a test for createInstanceState

Reply:
Done